timelineview/src/main/java/me/jerryhanks/timelineview/IndicatorAdapter.kt

```diff
@@ -101,4 +104,17 @@ class IndicatorAdapter<in T : TimeLine>(private val timeLines: MutableList<T>, p
         this.timeLines.addAll(items)
         notifyDataSetChanged()
     }
+
```

Comment:
when insert the data to the top, it can invoke notifyItemInserted(0) to update, but the the line of the second item is wrong, so need to update the second item.